tensornetwork/backends/abstract_backend.py

```diff
@@ -1022,3 +1022,11 @@ def item(self, tensor) -> Union[float, int, complex]:
       The value in tensor.
     """
     raise NotImplementedError("Backend {self.name} has not implemented item")
+
+  def cholesky_decomposition(self, 
+                             tensor: Tensor,
+                             pivot_axis: int = -1,
```

Comment:
Let's just call it "cholesky" like in numpy

Reply:
Very well, I will implement it right now!